src/drivers/pmw3901/pmw3901.cpp

```diff
@@ -77,18 +77,6 @@
 #include <board_config.h>
 
 /* Configuration Constants */
-#ifdef PX4_SPI_BUS_EXPANSION
-#define PMW3901_BUS PX4_SPI_BUS_EXPANSION
-#else
-#define PMW3901_BUS 0
-#endif
-
-#ifdef PX4_SPIDEV_EXPANSION_2
-#define PMW3901_SPIDEV PX4_SPIDEV_EXPANSION_2
```

Comment:
Is there a sensible default we can use? I'd like to avoid adding `PX4_SPI_BUS_BITCRAZE_FLOW` to all board configs that want to support the driver.
If there is none, we should add something like this:
```
#ifndef PX4_SPI_BUS_BITCRAZE_FLOW
#error "board_config.h needs to define PX4_SPI_BUS_BITCRAZE_FLOW"
#endif
```